### PR TITLE
Support 1.6.0+

### LINKED ISSUE #699: Backward compatibility with labels
Right now, Swarm will fail when using docker < 1.5.

We should revert to environment variables when using older versions.

### PR DESCRIPTION
**Pull request by vieux**
Fix #699

### CODE REVIEW

**Comment by vieux** [comment]
@moxiegirl can you please take a look at the doc change ?

**Comment by vieux** [comment]
@aluzzardi I decided to keep 1.6.0 for now.

**Comment by aluzzardi** [comment]
LGTM

**Comment by aluzzardi** [comment]
We should put a compatibility matrix somewhere

@moxiegirl, WDYT ?

**Comment by vieux** [comment]
tumblr_lstbslosfm1qd5bcwo1_500

**Comment by chanwit** [comment]
:+1:

File: docs/install-manual.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ This section tells you how to create a Docker Swarm on your network to use only
 You install Docker Swarm on a single system which is known as your Docker Swarm manager. You create the cluster, or swarm, on one or more additional nodes on your network. Each node in your swarm must:
 
 * be accessible by the swarm manager across your network
-* a TCP port open to listen for the swarm manager 
+* Docker Engine 1.6.0+ installed listenning on an open TCP port for the swarm manager
```
Comment (moxiegirl) [inline docs/install-manual.md]: Hi Vieux, Spelling error.  I'd like to split the second bullet into two.  Each point starts with a verb so:

* be accessible by the swarm manager across your network
* have Docker Engine 1.6.0+ installed
* open a TCP port to listen for the manager

Comment (vieux) [inline docs/install-manual.md]: ok

Comment (vieux) [inline docs/install-manual.md]: done

**Comment by vieux** [comment]
@moxiegirl should we merge it as is and add the `compatibility matrix ` afterwards

**Pull request merged**